presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -376,6 +376,18 @@ void PrestoServer::run() {
             .sendWithEOM();
       });
 
+  if (systemConfig->prestoNativeSidecar()) {
+    sessionPropertyReporter_ = std::make_unique<SessionPropertyReporter>();
+    httpServer_->registerGet(
+        "/v1/sessionProperties",
```

Comment:
This should likely be a different name. CamelCase isn't best practice for restful APIs. Maybe `v1/session-properties`? Or if we had other GET for properties - then we could do `v1/properties/session`.